eth/ethconsensusconfig/config.go

```diff
@@ -73,9 +77,24 @@ func CreateConsensusEngine(chainConfig *chain.Config, logger log.Logger, config
 			eng = parlia.New(chainConfig, db.OpenDatabase(consensusCfg.DBPath, logger, consensusCfg.InMemory, readonly), snapshots, chainDb[0])
 		}
 	case *chain.BorConfig:
+		genesisContractsClient := contract.NewGenesisContractsClient(chainConfig, chainConfig.Bor.ValidatorContract, chainConfig.Bor.StateReceiverContract)
```

Comment:
We might need some extra checks before this step. See [this](https://github.com/maticnetwork/bor/blob/develop/eth/ethconfig/config.go#L250-L253) for reference.